commit 3a4562a02856ceca516bb61945aec3a83b8826db
Author: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date:   Thu Feb 19 12:40:58 2015 -0600

    thermal/intel_powerclamp: add id for Avoton SoC
    
    Enable Intel Powerclamp driver on Atom* Processor C2000 Product
    Family for Microservers (Avoton). Avoton - SoCs for micro-servers
    has package C-states which can be used for idle injection.
    
    Reported-by: Jose Navarro <jose.navarro@intel.com>
    Suggested-by: Jacob Pan <jacob.jun.pan@linux.intel.com>
    Tested-by:  Jose Carlos Venegas Munoz <jos.c.venegas.munoz@intel.com>
    Signed-off-by: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
    Signed-off-by: Zhang Rui <rui.zhang@intel.com>

diff --git a/drivers/thermal/intel_powerclamp.c b/drivers/thermal/intel_powerclamp.c
index 6ceebd659dd4..12623bc02f46 100644
--- a/drivers/thermal/intel_powerclamp.c
+++ b/drivers/thermal/intel_powerclamp.c
@@ -688,6 +688,7 @@ static const struct x86_cpu_id intel_powerclamp_ids[] = {
 	{ X86_VENDOR_INTEL, 6, 0x45},
 	{ X86_VENDOR_INTEL, 6, 0x46},
 	{ X86_VENDOR_INTEL, 6, 0x4c},
+	{ X86_VENDOR_INTEL, 6, 0x4d},
 	{ X86_VENDOR_INTEL, 6, 0x56},
 	{}
 };